docs/release-notes/release-0-9-1.md

```diff
@@ -0,0 +1,97 @@
+# v0.9.1
+
+Release date: August 02, 2024
```

Comment:
Missing highlights

Reply:
> Missing highlights or main changes summary.

fixed in #[2603ec9](https://github.com/GreptimeTeam/docs/pull/1120/commits/2603ec9de98073357f2a134e0cd9849e18f17236). Let me also update the release SOP doc.